.github/workflows/pkgcheck.yml

```diff
@@ -8,14 +8,37 @@ jobs:
       fail-fast: false
       matrix:
         name: [
-          Ubuntu GCC
+          Ubuntu GCC,
+          Ubuntu GCC AARCH64,
+          Ubuntu GCC PPC
         ]
         include:
           - name: Ubuntu GCC
             os: ubuntu-latest
             compiler: gcc
             packages: ninja-build
 
+          - name: Ubuntu GCC AARCH64
+            os: ubuntu-latest
+            chost: aarch64-linux-gnu
+            compiler: aarch64-linux-gnu-gcc
+            cmake-args: -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-aarch64.cmake
+            packages: ninja-build qemu gcc-aarch64-linux-gnu libc-dev-arm64-cross
```

Comment:
`libc6-dev-arm64-cross` here?

Reply:
I used same package name as in cmake.yml.  Evidently the one missing the 6 is an alias, and it pulls in the right package.  Both files probably should use the real package name.